<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Keypad</class>
 <widget class="QWidget" name="Keypad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="obutton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>35</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tbutton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>35</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fstop">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>35</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fbutton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>92</y>
     <width>35</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="wbutton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>35</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="donebutton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>93</y>
     <width>66</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bpbutton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>76</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="numbutton_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>140</y>
     <width>35</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="numbutton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>35</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nbutton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>35</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ubutton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>35</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lbutton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>92</y>
     <width>35</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cbutton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>35</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rbutton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>35</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="donebutton_2">
   <property name="geometry">
    <rect>
     <x>436</x>
     <y>190</y>
     <width>35</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dbutton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>92</y>
     <width>35</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mbutton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>35</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="comma">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>35</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="apostrophe">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>92</y>
     <width>35</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bbutton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>35</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ibutton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>35</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="numbutton_4">
   <property name="geometry">
    <rect>
     <x>398</x>
     <y>190</y>
     <width>35</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 255);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ybutton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>35</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>10</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="jbutton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>92</y>
     <width>35</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="kbutton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>92</y>
     <width>35</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="vbutton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>35</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>198</x>
     <y>10</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Use Keypad to enter new value</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hbutton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>92</y>
     <width>35</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="question">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>35</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="numbutton_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>35</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 255);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="xbutton">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>140</y>
     <width>35</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="spacebutton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>233</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);

border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zbutton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>35</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gbutton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>92</y>
     <width>35</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qbutton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>35</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="abutton">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>92</y>
     <width>35</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbutton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>35</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sbutton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>92</y>
     <width>35</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ebutton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>35</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/empty.png);
color: rgb(255, 255, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
